components/RenderProposalContentLambda.tsx

```diff
@@ -534,6 +534,10 @@ export const labelOfProposalContentLambda = (
     return `Remove signer${content.removeOwners.length > 1 ? "s" : ""}`;
   } else if ("transfer" in content) {
     return `Transfer ${mutezToTez(content.transfer.amount)} Tez`;
+  } else if ("add_or_update_metadata" in content) {
+    return "Update Metadata(TZIP16)";
+  } else if ("proof_of_event" in content) {
+    return "Message Signing in Proof of event challenge(TZIP27)";
```

Comment:
Do we really want to put the Tzip number here?
If so, let's just add a space between the word and the parenthesis